app/components/satis/menu_item/component.html.slim

```diff
@@ -1,4 +1,4 @@
-li.rounded-sm.px-3.py-1.hover:bg-gray-100.dark:hover:bg-gray-200.flex.items-center data-controller="satis-menu" data-action="mouseover->satis-menu#show mouseleave->satis-menu#hide"
+li.rounded-sm.px-3.py-1.hover:bg-gray-100.dark:hover:bg-gray-200.flex.items-center data-controller="satis-menu" data-action="mouseenter->satis-menu#show"
```

Comment:
This will make submenu's also not close with mouseleave right?
Possibly make that optional / config /choice and put it behind the same new click-outside logic.

Reply:
I did this and along the way i found out why the filter was hiding. it was due to the loading overlay. 
i did not remove what i created. we still now can add a flag if it should close on mouse leave or click